assets/global.js

```diff
@@ -417,6 +418,11 @@ class HeaderDrawer extends MenuDrawer {
     trapFocus(this.mainDetailsToggle, summaryElement);
     document.body.classList.add(`overflow-hidden-${this.dataset.breakpoint}`);
   }
+
+  closeMenuDrawer(event, elementToFocus = false) {
```

Comment:
Don't think we need to set a default for `elementToFocus`, since we're only passing this param to the parent method which already sets a default?

```suggestion
  closeMenuDrawer(event, elementToFocus) {
```